src/xenia/base/system_gnulinux.cc

```diff
@@ -29,7 +33,19 @@ void LaunchWebBrowser(const std::string_view url) {
   system(cmd.c_str());
 }
 
-void LaunchFileExplorer(const std::filesystem::path& path) { assert_always(); }
+void LaunchFileExplorer(const std::filesystem::path& path) {
+  pid_t pid;
+  int n = path.string().length();
+  char cmdxdg[] = "xdg-open";
+  char cmdpath[n + 1];
+  char* argv[] = {cmdxdg, cmdpath, NULL};
+  int status = -1;
```

Comment:
Declaring this variable upfront is not required as it's first assigned to in the same scope.